src/backend/joanie/core/serializers/client.py

```diff
@@ -1041,6 +1041,7 @@ class OrderPaymentSerializer(serializers.Serializer):
         min_value=D(0.00),
         required=True,
     )
+    amount_currency = serializers.SerializerMethodField(read_only=True)
```

Comment:
Why `amount_currency` and not only `currency` ?

Reply:
:heavy_plus_sign:

Reply:
That's because we are using this naming in other serializers